examples/chip-tool/commands/common/CommandInvoker.h

```diff
@@ -211,12 +222,13 @@ CHIP_ERROR InvokeCommand(DeviceProxy * aDevice, void * aContext,
 template <typename RequestType, typename std::enable_if_t<!RequestType::MustUseTimedInvoke(), int> = 0>
 CHIP_ERROR InvokeGroupCommand(DeviceProxy * aDevice, void * aContext,
                               typename detail::CommandInvoker<RequestType>::SuccessCallback aSuccessCallback,
-                              typename detail::CommandInvoker<RequestType>::FailureCallback aFailureCallback, GroupId groupId,
+                              typename detail::CommandInvoker<RequestType>::FailureCallback aFailureCallback,
+                              typename detail::CommandInvoker<RequestType>::DoneCallback aDoneCallback, GroupId groupId,
                               const RequestType & aRequestData)
 {
-    auto invoker = detail::CommandInvoker<RequestType>::Alloc(aContext, aSuccessCallback, aFailureCallback);
+    auto invoker = detail::CommandInvoker<RequestType>::Alloc(aContext, aSuccessCallback, aFailureCallback, aDoneCallback);
     VerifyOrReturnError(invoker != nullptr, CHIP_ERROR_NO_MEMORY);
-    ReturnErrorOnFailure(invoker->InvokeGroupCommand(aDevice, groupId, aRequestData));
+    ReturnErrorOnFailure(invoker->InvokeGroupCommand(aDevice->GetExchangeManager(), groupId, aRequestData));
     invoker.release();
```

Comment:
Should document that at this point `invoker` will be deleted already.

Reply:
done